storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -277,6 +277,12 @@
     @IsBoolean
     public static final String TOPOLOGY_ENABLE_V2_METRICS_TICK = "topology.enable.v2.metrics.tick";
 
+    /**
+     * This config allows V2 metrics reporting through the metrics tick to use short names.
+     */
+    @IsBoolean
+    public static final String TOPOLOGY_METRICS_USE_SHORTNAME = "topology.metrics.use.shortname";
```

Comment:
We don't really need this config. 
The taskIdMapping is only used for v2.metrics.tick. We save short name in the taskIdMapping to conform with V1 method of sending metrics.

Metrics register at codehale metricRegistry should just use the long time.

Reply:
I'm fine with removing the config.  @kishorvpatil - are you also ok with removing this option and always reporting short names?

Reply:
I think having this configuration, allows us use shortnames even while topology is not using v2 metrics tick. So we should use this configuration.

Reply:
This PR will only support reporting short names for the V2 metrics tick.  We would need a different solution for tracking short metrics names for the non-V2 tick.

Reply:
We can't really use shortname if it is using v2 metrics because codehale metric registry maps name to metric object.

Reply:
okay.